src/components/DragWindowRegion.tsx

```diff
@@ -9,17 +9,20 @@ interface DragWindowRegionProps {
   title?: ReactNode;
 }
 
+// Check if running on macOS
+const isMacOS = typeof window !== "undefined" && window.navigator.platform.includes("Mac");
+
 export default function DragWindowRegion({ title }: DragWindowRegionProps) {
   return (
     <div className="flex w-screen items-stretch justify-between">
-      <div className="draglayer w-full">
+      <div className={`draglayer w-full ${isMacOS ? "pl-20" : ""}`}>
```

Comment:
Use [`cn`](https://github.com/LuanRoger/electron-shadcn/blob/main/src/utils/tailwind.ts) to merge the Tailwind classes